Code/Framework/AzToolsFramework/Tests/Prefab/PrefabDeleteAsOverrideTests.cpp

```diff
@@ -279,4 +279,90 @@ namespace UnitTest
         ValidateEntityUnderInstance(secondCarInstance->get().GetContainerEntityId(), tireEntityAlias, tireEntityName);
         ValidateNestedInstanceUnderInstance(secondCarInstance->get().GetContainerEntityId(), childInstanceAlias);
     }
+
+    TEST_F(PrefabDeleteAsOverrideTests, DeleteEntitiesAndAllDescendantsInInstance_FocusOnDeletedPrefabFromRootSucceeds)
+    {
+        // Level            <-- deletes Wheel instance as an override
+        // | Car            <-- focuses on Car to make Wheel available
+        //   | Wheel        <-- focuses on Wheel
+        //     | Tire
+
+        const AZStd::string carPrefabName = "CarPrefab";
+        const AZStd::string wheelPrefabName = "WheelPrefab";
+        const AZStd::string tireEntityName = "Tire";
+
+        AZ::IO::Path engineRootPath;
+        m_settingsRegistryInterface->Get(engineRootPath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
+        AZ::IO::Path carPrefabFilepath(engineRootPath);
+        carPrefabFilepath.Append(carPrefabName);
+        AZ::IO::Path wheelPrefabFilepath(engineRootPath);
+        wheelPrefabFilepath.Append(wheelPrefabName);
```

Comment:
Nit: I think you could consolidate this a little like so:
```suggestion
        AZ::IO::Path carPrefabFilepath = engineRootPath / carPrefabName;
        AZ::IO::Path wheelPrefabFilepath = engineRootPath / wheelPrefabName;
```